materials/material.py

```diff
@@ -0,0 +1,57 @@
+"""Representation of a material's engineering properties and other data."""
+import yaml
+from materials.property import Property, StateDependentProperty
+
+class Material:
+    """An engineering material, in a particular form and condition."""
+    def __init__(self, name, form=None, condition=None, category=None, subcategory=None,
+                 references=None, properties_dict=None):
```

Comment:
[taste] I might recommend comments about what form,condition,etc are permitted to be. Are they classes? Are they duck-types with some expected properties?

Reply:
fixed in https://github.com/mvernacc/material-properties-interchange/pull/1/commits/8231cb717171b9b22b75e592594a69c1ac5f5361